en/getting_started/basic_flight_modes.md

```diff
@@ -2,185 +2,178 @@
 
-   **• Manual Modes**
+This topic provide a very high level view of each of the flight modes, and the (mostly minor) differences in how they behave in copter, fixed-wing and VTOL frames. For more detailed information see [Flight Modes](../flight_modes/README.md).
 
-   **•	Assisted Modes**
+> **Tip** ![GPS](../../images/flight_modes/GPS_s.png) - This symbol is used to denote modes that *require* position information! These modes cannot be enabled without a position lock from GPS, VIO, or some other positioning system.
 
-   **•	Autonomous Modes**
+## Copter
 
-You have to remember that the flight modes have a slightly different way of behavior depending on the air-frame. There are differences between Airplane, Multicopter and VTOL modes so we will explain it in brief on this page.
+### Manual Modes
 
-Some modes are GPS guided (dependent) and we will use ![GPS](../../images/flight_modes/GPS_s.png) symbol to denote them. In order to function properly and be enabled, this GPS dependent mode’s need a GPS lock and they cannot be enabled during flight without GPS Lock. 
+Manual modes are those where the user has control over vehicle movement via the RC control sticks (or joystick). Vehicle movement always follows stick movement, but the level/type of response changes depending on the mode. For example, experienced fliers can use modes that provide direct passthrough of stick positions to actuators, while beginners will often choose modes that are less responsive to sudden stick-position changes.
 
-> **Warning** It is highly advisable when you start  your aircraft at the flight field, to wait for GPS Lock prior arming to have a good Home Point Location Lock. You could see it when the Pixhawk LED starts blinking green or as a "H" symbol apears in the QGroundControl. Without Home Point GPS Lock it is impossible for the **Return to Launch / Return to Home** to be engaged and the craft will never come back in case of emergency.
-You can read in depth about Flight Modes in the Flight Mode Setup page.
+#### Manual/Stabilized Mode
 
+In Stabilized flight your copter will hover in level flight once the Pitch and Roll sticks are at center position. You will have a full manual angle control over the copter. If your craft is perfectly set and balanced it will hold the altitude if throttle is at 50% in the middle of the scale at calm weather. If there is a wind blowing the craft will drift in the direction of the wind and you have to control the throttle all the time during wind gusts to hold altitude. In order to perform a maneuver, you have to hold your Pitch, Roll and Yaw command, otherwise the craft will level itself the moment you release the control sticks.
 
+![MC Manual Flight](../../images/flight_modes/manual_stabilized_MC.png)
 
-## COPTER FLIGHT MODES
-
-## Manual Modes
+#### Acro (Acrobatic Mode)
```

Comment:
@tubeme 
- All headings are now nested to right levels
- Note that as standard we use the names for the modes "as they appear in QGC". So this is "Acro", not Acrobatic or ACROBATIC or whatever. For consistency I will  change this to `#### Acro Mode (Acrobatic)`

Reply:
Perfect... thanks